[0m09:23:05.872294 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E9738A1BE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E976364C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E976364E50>]}


============================== 09:23:05.934787 | 57d0bebb-01dc-47d9-805d-af37a9fc6c4d ==============================
[0m09:23:05.934787 [info ] [MainThread]: Running with dbt=1.8.6
[0m09:23:05.934787 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init superside_hr', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:23:06.744925 [debug] [MainThread]: Starter project path: C:\Users\hp\anaconda3\lib\site-packages\dbt\include\starter_project
[0m09:23:06.899964 [info ] [MainThread]: 
Your new dbt project "superside_hr" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m09:23:06.903980 [info ] [MainThread]: Setting up your profile.
[0m10:11:24.721392 [info ] [MainThread]: Profile superside_hr written to C:\Users\hp\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m10:11:24.741135 [debug] [MainThread]: Command `dbt init` succeeded at 10:11:24.740253 after 2899.01 seconds
[0m10:11:24.743135 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E9738A1BE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E976339130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E976328A30>]}
[0m10:11:24.746674 [debug] [MainThread]: Flushing usage events
[0m10:11:47.301985 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001AD07431C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001AD07DB93D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001AD070F5E50>]}


============================== 10:11:47.301985 | 380b65ae-6a13-477e-9c53-32286d7c2332 ==============================
[0m10:11:47.301985 [info ] [MainThread]: Running with dbt=1.8.6
[0m10:11:47.301985 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:11:47.463035 [info ] [MainThread]: dbt version: 1.8.6
[0m10:11:47.465034 [info ] [MainThread]: python version: 3.9.19
[0m10:11:47.467035 [info ] [MainThread]: python path: C:\Users\hp\anaconda3\python.exe
[0m10:11:47.468032 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m10:11:50.638163 [info ] [MainThread]: Using profiles dir at C:\Users\hp\.dbt
[0m10:11:50.638163 [info ] [MainThread]: Using profiles.yml file at C:\Users\hp\.dbt\profiles.yml
[0m10:11:50.638163 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
[0m10:11:50.638163 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m10:11:50.654933 [debug] [MainThread]: Command `dbt debug` failed at 10:11:50.654933 after 3.50 seconds
[0m10:11:50.655934 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001AD07431C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001AD09BD5A90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001AD070F5C10>]}
[0m10:11:50.656932 [debug] [MainThread]: Flushing usage events
[0m11:20:09.108659 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029AA86A1C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029AAB164B50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029AAB164D60>]}


============================== 11:20:09.108659 | 440fe69a-7d9f-49fb-a5c5-8bf73340a81e ==============================
[0m11:20:09.108659 [info ] [MainThread]: Running with dbt=1.8.6
[0m11:20:09.108659 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt test', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:20:09.124288 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:20:09.124288 [debug] [MainThread]: Command `dbt test` failed at 11:20:09.124288 after 0.12 seconds
[0m11:20:09.124288 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029AA86A1C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029AAB164BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029AAB164AC0>]}
[0m11:20:09.124288 [debug] [MainThread]: Flushing usage events
[0m11:21:27.610454 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E33DF01BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E3409C03A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E3409C0130>]}


============================== 11:21:27.626093 | ec8d90f3-dbf4-442a-b46c-e1ce20380d1c ==============================
[0m11:21:27.626093 [info ] [MainThread]: Running with dbt=1.8.6
[0m11:21:27.626093 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'invocation_command': 'dbt run', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:21:27.626093 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:21:27.639075 [debug] [MainThread]: Command `dbt run` failed at 11:21:27.639075 after 0.12 seconds
[0m11:21:27.641079 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E33DF01BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E34098A370>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E34098A760>]}
[0m11:21:27.642668 [debug] [MainThread]: Flushing usage events
[0m11:27:14.832662 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185C16F1C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185C41B5BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185C41B5D90>]}


============================== 11:27:14.848311 | 65059ccb-49f0-431b-887d-4a5e596faddb ==============================
[0m11:27:14.848311 [info ] [MainThread]: Running with dbt=1.8.6
[0m11:27:14.848311 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt test', 'send_anonymous_usage_stats': 'True'}
[0m11:27:14.848311 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:27:14.848311 [debug] [MainThread]: Command `dbt test` failed at 11:27:14.848311 after 0.13 seconds
[0m11:27:14.848311 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185C16F1C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185C41B5C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185C41B5B20>]}
[0m11:27:14.848311 [debug] [MainThread]: Flushing usage events
[0m09:09:54.812847 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021344BB1C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021347674B50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021347674D60>]}


============================== 09:09:54.844149 | b0a1cd66-91ce-4db5-b252-237e4802f4f6 ==============================
[0m09:09:54.844149 [info ] [MainThread]: Running with dbt=1.8.6
[0m09:09:54.844149 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m09:09:54.844149 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:09:54.859701 [debug] [MainThread]: Command `dbt run` failed at 09:09:54.859701 after 0.25 seconds
[0m09:09:54.859701 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021344BB1C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021347674BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021347674AC0>]}
[0m09:09:54.859701 [debug] [MainThread]: Flushing usage events
[0m09:29:04.146385 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204AC621C70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204AF0E6C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204AF0E6DF0>]}


============================== 09:29:04.146385 | 8249e43c-2b21-458f-800c-1d4251f4cc29 ==============================
[0m09:29:04.146385 [info ] [MainThread]: Running with dbt=1.8.6
[0m09:29:04.162009 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m09:29:04.162009 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:29:04.170805 [debug] [MainThread]: Command `dbt run` failed at 09:29:04.170805 after 0.25 seconds
[0m09:29:04.172808 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204AC621C70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204AF0E6C70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204AF0E6B80>]}
[0m09:29:04.174840 [debug] [MainThread]: Flushing usage events
[0m15:50:59.319603 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BFA2D21C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BFA57E0460>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BFA57E0130>]}


============================== 15:50:59.397700 | cf3a62d1-3d0d-4c24-a612-6222bacf01d0 ==============================
[0m15:50:59.397700 [info ] [MainThread]: Running with dbt=1.8.6
[0m15:50:59.397700 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt deps', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:50:59.397700 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:50:59.397700 [debug] [MainThread]: Command `dbt deps` failed at 15:50:59.397700 after 0.29 seconds
[0m15:50:59.413348 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BFA2D21C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BFA57BD9D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BFA57BD970>]}
[0m15:50:59.414798 [debug] [MainThread]: Flushing usage events
[0m16:27:07.993573 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D14A121BE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D14CBE1B50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D14CBE1FA0>]}


============================== 16:27:08.103011 | 59d0c55a-8841-407b-bb3f-b8fb46ce2f56 ==============================
[0m16:27:08.103011 [info ] [MainThread]: Running with dbt=1.8.6
[0m16:27:08.103011 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'C:\\Users\\hp\\Desktop\\Superside_HR_Manager\\logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m16:28:38.441940 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '59d0c55a-8841-407b-bb3f-b8fb46ce2f56', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D151648580>]}
[0m16:28:38.551287 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '59d0c55a-8841-407b-bb3f-b8fb46ce2f56', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D14C044880>]}
[0m16:28:38.551287 [info ] [MainThread]: Registered adapter: bigquery=1.8.2
[0m16:28:38.577068 [debug] [MainThread]: checksum: 2f2fb10c73cafe3e79aa104295a8244291492b32b5598ce46af770f7d0bd5194, vars: {}, profile: , target: , version: 1.8.6
[0m16:28:38.578323 [info ] [MainThread]: Unable to do partial parsing because saved manifest not found. Starting full parse.
[0m16:28:38.578323 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'partial_parser', 'label': '59d0c55a-8841-407b-bb3f-b8fb46ce2f56', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D149DF6DF0>]}
[0m16:28:41.508462 [warn ] [MainThread]: [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.superside_hr.staging
[0m16:28:41.522524 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': '59d0c55a-8841-407b-bb3f-b8fb46ce2f56', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D1528C4EB0>]}
[0m16:28:41.717696 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': '59d0c55a-8841-407b-bb3f-b8fb46ce2f56', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D1529B7430>]}
[0m16:28:41.717696 [info ] [MainThread]: Found 476 macros
[0m16:28:41.717696 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': '59d0c55a-8841-407b-bb3f-b8fb46ce2f56', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D152886460>]}
[0m16:28:41.717696 [info ] [MainThread]: 
[0m16:28:41.717696 [warn ] [MainThread]: Nothing to do. Try checking your model configs and model specification args
[0m16:28:41.730068 [debug] [MainThread]: Command end result
[0m16:28:41.787275 [debug] [MainThread]: Command `dbt run` succeeded at 16:28:41.787275 after 94.09 seconds
[0m16:28:41.787275 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D14A121BE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D14C512E20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002D15164DD60>]}
[0m16:28:41.787275 [debug] [MainThread]: Flushing usage events
[0m16:29:47.218012 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200EEA21BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200F03A7730>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200EE70CC70>]}


============================== 16:29:47.218012 | 66d95c74-f235-4f40-a543-3b94f97b2b3d ==============================
[0m16:29:47.218012 [info ] [MainThread]: Running with dbt=1.8.6
[0m16:29:47.233636 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\hp\\.dbt', 'log_path': 'C:\\Users\\hp\\Desktop\\Superside_HR_Manager\\logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:29:47.323569 [info ] [MainThread]: dbt version: 1.8.6
[0m16:29:47.323569 [info ] [MainThread]: python version: 3.9.19
[0m16:29:47.339178 [info ] [MainThread]: python path: C:\Users\hp\anaconda3\python.exe
[0m16:29:47.339178 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m16:29:50.863810 [info ] [MainThread]: Using profiles dir at C:\Users\hp\.dbt
[0m16:29:50.863810 [info ] [MainThread]: Using profiles.yml file at C:\Users\hp\.dbt\profiles.yml
[0m16:29:50.879409 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\hp\Desktop\Superside_HR_Manager\dbt_project.yml
[0m16:29:50.879409 [info ] [MainThread]: adapter type: bigquery
[0m16:29:50.879409 [info ] [MainThread]: adapter version: 1.8.2
[0m16:29:51.046490 [info ] [MainThread]: Configuration:
[0m16:29:51.046490 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m16:29:51.062094 [info ] [MainThread]:   dbt_project.yml file [[32mOK found and valid[0m]
[0m16:29:51.062094 [info ] [MainThread]: Required dependencies:
[0m16:29:51.062094 [debug] [MainThread]: Executing "git --help"
[0m16:29:51.229550 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m16:29:51.229550 [debug] [MainThread]: STDERR: "b''"
[0m16:29:51.229550 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m16:29:51.229550 [info ] [MainThread]: Connection:
[0m16:29:51.229550 [info ] [MainThread]:   method: oauth
[0m16:29:51.229550 [info ] [MainThread]:   database: glass-mantra-128509
[0m16:29:51.241054 [info ] [MainThread]:   execution_project: glass-mantra-128509
[0m16:29:51.242053 [info ] [MainThread]:   schema: superside_hr
[0m16:29:51.244926 [info ] [MainThread]:   location: EU
[0m16:29:51.246925 [info ] [MainThread]:   priority: interactive
[0m16:29:51.251225 [info ] [MainThread]:   maximum_bytes_billed: None
[0m16:29:51.253326 [info ] [MainThread]:   impersonate_service_account: None
[0m16:29:51.254220 [info ] [MainThread]:   job_retry_deadline_seconds: None
[0m16:29:51.256505 [info ] [MainThread]:   job_retries: 1
[0m16:29:51.259481 [info ] [MainThread]:   job_creation_timeout_seconds: None
[0m16:29:51.261477 [info ] [MainThread]:   job_execution_timeout_seconds: 300
[0m16:29:51.262978 [info ] [MainThread]:   timeout_seconds: 300
[0m16:29:51.265886 [info ] [MainThread]:   client_id: None
[0m16:29:51.265886 [info ] [MainThread]:   token_uri: None
[0m16:29:51.265886 [info ] [MainThread]:   dataproc_region: None
[0m16:29:51.273227 [info ] [MainThread]:   dataproc_cluster_name: None
[0m16:29:51.274945 [info ] [MainThread]:   gcs_bucket: None
[0m16:29:51.276412 [info ] [MainThread]:   dataproc_batch: None
[0m16:29:51.278451 [info ] [MainThread]: Registered adapter: bigquery=1.8.2
[0m16:29:51.282667 [debug] [MainThread]: Acquiring new bigquery connection 'debug'
[0m16:29:51.282667 [debug] [MainThread]: Opening a new connection, currently in state init
[0m16:30:03.620124 [debug] [MainThread]: BigQuery adapter: Got an error when attempting to create a bigquery client: 'Runtime Error
  Failed to authenticate with supplied credentials
  error:
  Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.'
[0m16:30:03.620124 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m16:30:03.620124 [info ] [MainThread]: [31m1 check failed:[0m
[0m16:30:03.620124 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  Runtime Error
    Failed to authenticate with supplied credentials
    error:
    Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m16:30:03.632014 [debug] [MainThread]: Command `dbt debug` failed at 16:30:03.631011 after 16.56 seconds
[0m16:30:03.632014 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m16:30:03.634495 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200EEA21BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200F60D2E20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200F60D2970>]}
[0m16:30:03.635779 [debug] [MainThread]: Flushing usage events
